application/models/organization_test.go

```diff
@@ -2,26 +2,13 @@ package models
 
 import (
 	"github.com/gobuffalo/nulls"
-	"github.com/gofrs/uuid"
 	"github.com/silinternational/handcarry-api/domain"
 
 	"testing"
 )
 
 func TestFindOrgByUUID(t *testing.T) {
-	// Load Organization test fixtures
-	orgUuidStr := "51b5321d-2769-48a0-908a-7af1d15083e2"
-	orgUuid1, _ := uuid.FromString(orgUuidStr)
-	org := Organization{
-		Name:       "ACME",
-		Uuid:       orgUuid1,
-		AuthType:   "saml2",
-		AuthConfig: "[]",
-	}
-	if err := DB.Create(&org); err != nil {
-		t.Errorf("could not run test ... %v", err)
-		t.FailNow()
-	}
+	org, _ := createOrgFixtures(t)
```

Comment:
I tried to keep tests atomic/independent. If you see one that's not, I'll try to fix it.

Reply:
Personally, I'm OK with it, but there are two tests that call `createOrgFixtures()`

Reply:
Oh, I see. I was thinking differently. My main concern was the _content_ of the database: making certain that it's cleaned up after each test so that you don't key unique constraint violations and such. This is another concern, but definitely a tradeoff with code duplication.